functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/failed/ReportsVerifier.java

```diff
@@ -0,0 +1,22 @@
+package org.arquillian.smart.testing.ftest.failed;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+import org.assertj.core.api.Assertions;
+
+import static org.arquillian.smart.testing.spi.TestResult.TEMP_REPORT_DIR;
+
+public class ReportsVerifier {
+
+    static void assertNoReportsDirectoryPresent(Project project) throws IOException {
```

Comment:
How about `reportsDirectoryDoesNotExists`?

Reply:
Let's leave it like it is for now. I will create a separated issue to clean up the assertions using `assertj` as we brought similar static methods in other tests.

Reply:
Related issue #140